### PR TITLE
bucketlist detail

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/NSNG-260

### CODE REVIEW

File: resources/js/views/Collection.vue [hunk 1]
```diff
@@ -1,17 +1,20 @@
 <template>
-    <div class="p-4">
-        <Timeline />
+    <div>
+        <Bucketlist :id="bucketlistId" />
```
Comment (igor-kamil) [inline resources/js/views/Collection.vue:3]: add `v-if` it exists.

File: resources/js/components/Bucketlist.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<template>
+    <article v-if="bucketlist">
+        <div class="bg-green/20 p-4">
+            <h2 class="mt-2 text-xl font-medium leading-6">{{ $t('Hidden artworks') }}</h2>
+            <p class="mt-4 text-3xl font-bold leading-8">{{ found.length }}/{{ bucketlist.items.length }} collected</p>
```
Comment (igor-kamil) [inline resources/js/components/Bucketlist.vue:5]:
```suggestion
            <p class="mt-4 text-3xl font-bold leading-8">{{ found.length }}/{{ bucketlist.items.length }} {{ $t('collected') }}</p>
```

File: resources/js/components/Bucketlist.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            <p class="mt-4">{{ $t('Nájdi všetky diela v zozname nižšie a odhalíš ich tajomstvo') }}</p>
```
Comment (igor-kamil) [inline resources/js/components/Bucketlist.vue:6]: English translation strings are the defaults...
well... doing the implementation of translations afterward is messy from my point of view. this should be already specified within the task... for the next time

**Review by igor-kamil — COMMENTED** [review]
implement the fallback, when there is no `VITE_DEFAULT_BUCKETLIST` set


bigger spacing between bucketlist and viewed items

Snímka obrazovky 2023-04-30 o 10 07 38

File: resources/js/components/Bucketlist.vue (hunk 3 above)
Comment (rastislav-chynoransky) [reply resources/js/components/Bucketlist.vue:6]: Yes, please!

**Comment by rastislav-chynoransky** [comment]
> implement the fallback, when there is no `VITE_DEFAULT_BUCKETLIST` set
> 
> bigger spacing between bucketlist and viewed items
> 
> Snímka obrazovky 2023-04-30 o 10 07 38

will fix the spacing in the timeline branch

**Pull request merged**